site/public/index.html

```diff
@@ -92,7 +92,7 @@
 			<div class="hero-banner">
 				<div class="l-padding">
 					<h1>Coronavirus (COVID&#8209;19)</h1>
-					<p>This site will link you to the latest Coronavirus news, updates and advice from government agencies across Australia</p>
+					<p>This site is regularly updated to give you to the latest Coronavirus news, updates and advice from government agencies across Australia.</p>
```

Comment:
```suggestion
					<p>This site is regularly updated to give you the latest Coronavirus news, updates and advice from government agencies across Australia.</p>
```

Reply:
Love the suggestion, but we'll save it for next release